packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -17,88 +17,121 @@
 import {types} from '@opencensus/opencensus-core';
 import {classes} from '@opencensus/opencensus-core';
 import {logger} from '@opencensus/opencensus-core';
+import * as http from 'http';
 import * as semver from 'semver';
 import * as shimmer from 'shimmer';
 import * as url from 'url';
 import * as uuid from 'uuid';
 
-// import * as Debug from 'debug';
-// const debug = Debug('opencensus');
 
+// TODO: maybe we should have a setup as a Client or as Server.
+
+type HttpModuleType = typeof http;
```

Comment:
Nit: `HttpModule` is fine, it's already a type. Also, call the module itself `httpModule` rather than `http`.

Reply:
Done!